### DISCUSSION TITLE
After downloading FBW experimental I can´t get any thrust

### DISCUSSION THREAD

**Discussion by bmelen** [flybywiresim/aircraft#3576] [Help]
What is wrong?

**Comment by hotshotp** [comment]
please be more specific, this doesn't help in identifying your issue

**Comment by bmelen** [comment]
On take off I set Thrust to Togo. There is no power and the thrust levers start to shake.

 

Från: hotshotp <notifications@github.com> 
Skickat: den 17 februari 2021 19:01
Till: flybywiresim/a32nx <a32nx@noreply.github.com>
Kopia: bmelen <bert.melen@gmail.com>; Author <author@noreply.github.com>
Ämne: Re: [flybywiresim/a32nx] After downloading FBW experimental I can´t get any thrust (#3576)

 

please be more specific, this doesn't help in identifying your issue
